power-control-x86/src/power_control.cpp

```diff
@@ -1804,6 +1804,8 @@ int main(int argc, char* argv[])
     {
         return -1;
     }
+    else
+	system("yaapd -v 4 &");
```

Comment:
1. This is not the right place to start yaapd. Power control is just initializing the Active Low GPIO to high. it wont power on the system.
2. We should not start it in debug mode.
3. 

It should be started using systemctl start yaapd and stopped using systemctl stop yaapd in the follwoing APIs.

static void powerOn()
{
    setGPIOOutputForMs("ASSERT_PWR_BTN_L", 0, powerPulseTimeMs);
}

static void gracefulPowerOff()
{
    setGPIOOutputForMs("ASSERT_PWR_BTN_L", 0, powerPulseTimeMs);
}

static void forcePowerOff()
{
    setGPIOOutputForMs("ASSERT_PWR_BTN_L", 0, forceOffPulseTimeMs);
    return;
}